src/showErrors.coffee

```diff
@@ -21,11 +21,11 @@ showErrorsModule.directive 'showErrors',
       showSuccess = getShowSuccess options
       trigger = getTrigger options
 
-      inputEl   = el[0].querySelector '.form-control[name]'
+      inputEl   = el[0].querySelector 'input[name],select[name],password[name],email[name],datetime[name],datetime-local[name],date[name],month[name],time[name],week[name],number[name],url[name],search[name],tel[name],color[name],textarea[name]'
```

Comment:
don't leave out the `.form-control[name]`, because there may be other inputs (custom elements) that behave like form controls that you are missing now.